src/scss/config/scale/_spacing-sizes.scss

```diff
@@ -0,0 +1,11 @@
+$gutter: 0.75rem;
+$newline--default: $gutter;
+$newline: $newline--default;
+$gutter-plus: calc(var(--gutter) * 1.5);
```

Comment:
I'm curious... We can reference a Sass variable like line 2 here or we can use the CSS variable like line 4. If we do the former, we get this type of visual preview which gets grouped together:
![Screenshot 2023-08-08 at 11 13 22 AM](https://github.com/oddbird/oddleventy/assets/1581694/1df87dd3-9cfb-497a-959f-a3ed02c2b559)

vs if we use the css custom properties we see this:
![Screenshot 2023-08-08 at 11 14 05 AM](https://github.com/oddbird/oddleventy/assets/1581694/7b64c9d2-950f-4a1d-9184-cfb771352882)

I know we are using the Sass variables on OddBooks. If we use the CSS custom properties, it will show the values as defined so for double-gutter it will show calc(var(--gutter) * 2) as opposed to the Sass variables which will calculate and show the result which is 1.5rem.

I don't know if there are other reasons or advantages (I know we discussed this a while back and this project was using a mix of both custom properties and Sass Variables so I thought I'd check to see if there was any opinions. I do like seeing both the math (showing the custom props) and the calculated value, which you get if we use the Sass Variables. I don't love referencing Sass variables from other files because of the namespacing, but that isn't a big deal. The maintainer in me like that the namespace tells you where it is coming from, but author in me doesn't love the few extra things to check/write ¯\_(ツ)_/¯

Reply:
I don't think we should make the decision based too much on how it looks in the style guide. I'd probably lean towards capturing relationships in custom properties. I think that's what you're suggesting as well? Especially for something like `newline` and `newline-default` - we want that relationship to exist in the CSS, so when you update the default, the other one also updates.

Reply:
I changed `newline` and `newline-default` back to custom properties to capture the sizing relationships as suggested in https://github.com/oddbird/oddleventy/pull/306/commits/6bf56ceeaf8da19b0c67605c9341586443c3d6ae